dss/storage/checkout/__init__.py

```diff
@@ -57,7 +59,7 @@ def touch_test_file(replica: Replica, dst_bucket: str) -> bool:
     :param replica: the replica to execute the checkout in.
     :return: True if able to write, if not raise DestinationBucketNotWritableError.
     """
-    test_object = "touch.txt"
+    test_object = f"touch_{''.join(random.choices(hexdigits, k=4))}.txt"
```

Comment:
pls change to `f"touch/{''.join(random.choices(hexdigits, k=4))}.txt"` as that will keep the bucket listing less messy.